docs/articles/infrastructure/ci-cd/freeipa.rst

```diff
@@ -0,0 +1,240 @@
+:orphan:
+
+FreeIPA
+=======
+
+FreeIPA is an identity management system which comes with an LDAP server, kerberos and a web UI built in. It has the ability to "Manage Linux users and client hosts in your realm from one central location with CLI, Web UI or RPC access." It can thus be used to centrally manage accounts and allow users to login to multiple servers and websites with the same credentials.
+
+Server setup
+------------
+
+This guide assumes you're running Ubuntu server 16:04 but should work for most major linux distributions.
+
+Prerequisites
+^^^^^^^^^^^^^
+
+Choose a domain to use for your deployment. It's important that you have control over it. Never use an example domain or a top domain which does not exist, it will only cause problems later on. It's okay to use a subdomain which is not public though.
+
+.. warning::
+
+    It's recommended to use a dedicated host or VM (not docker) for running FreeIPA, with a dedicated IPv4 address. FreeIPA server integrates into a lot of services on the host and running other services on the same host will eventually cause problems.
+
+It's important that the server has a fully qualified domain name. You can either set the hostname when you create the server or set it from the command line after the server is created, using the hostname command:
+
+.. code-block:: bash
+
+    hostname ipaserver.example.com
+
+Then set up ``/etc/hosts`` file with the fully qualified domain name pointing to the servers IP:
+
+.. code-block:: none
+    :caption: Example /etc/hosts
+
+    # Replace with your own values
+    198.51.100.5 ipaserver.example.com ipaserver
```

Comment:
If this is a real IP address, please use an example address or one from a reserved block (like 10.0.0.0/8)

Reply:
Actually I'm using a block reserved for documentation, see https://tools.ietf.org/html/rfc5737 for example IP addresses. To quote:

>The blocks 192.0.2.0/24 (TEST-NET-1), 198.51.100.0/24 (TEST-NET-2),
>   and 203.0.113.0/24 (TEST-NET-3) are provided for use in
>   documentation.

Reply:
Ah nice, I learned something new today!